src/main/java/createQuiz/CreateQuizInteracter.java

```diff
@@ -1,4 +1,36 @@
 package createQuiz;
 
-public class CreateQuizInteracter {
```

Comment:
Interactor spelled with an "o" not an "e". Lets just change to keep it consistent.